### PR TITLE
v0.6.6: Added python3 support

### PR DESCRIPTION
**Pull request by jamesba**

### CODE REVIEW

File: stdeb.cfg [hunk 1]
```diff
@@ -1,4 +1,6 @@
 [DEFAULT]
 Depends: etcd, python-gevent, python-flask, python-systemd, python-ws4py, python-requests, python-jsonschema
```
Comment (andrewbonney) [inline stdeb.cfg]:
```suggestion
Depends: etcd, python-gevent, python-flask, python-systemd, python-ws4py, python-requests, python-jsonschema, python-six
```

File: nmosregistration/etcd_backend.py [hunk 2]
```diff
@@ -12,15 +12,17 @@
...
+from __future__ import absolute_import
+
 from gevent import monkey
 monkey.patch_all()
 
 import requests
 import json
 import gevent
-import urllib
+from six.urllib.parse import urlencode
```
Comment (andrewbonney) [inline nmosregistration/etcd_backend.py]:
```suggestion
from six.moves.urllib.parse import urlencode
```

File: stdeb.cfg [hunk 3]
```diff
@@ -1,4 +1,6 @@
...
 Recommends: ips-reverseproxy-common, python-nmoscommon
+Depends3: etcd, python3-gevent, python3-flask, python3-systemd, python3-ws4py, python3-requests, python3-jsonschema
```
Comment (andrewbonney) [inline stdeb.cfg]:
```suggestion
Depends3: etcd, python3-gevent, python3-flask, python3-systemd, python3-ws4py, python3-requests, python3-jsonschema, python3-six
```

File: setup.py [hunk 4]
```diff
@@ -57,7 +57,7 @@ def find_packages(path, base=""):
 ]
 
 setup(name="registryaggregator",
-      version="0.6.5",
+      version="0.6.6",
```
Comment (andrewbonney) [inline setup.py:61]: Looks like setup.py also needs a 'six' dependency, but GitHub won't let me suggest in an unchanged bit unfortunately!

**Review by andrewbonney — COMMENTED** [review]
Just a few minor issues, but other than that it seems to do the job.

**Review by andrewbonney — APPROVED** [review]

**Review by simonrankine — APPROVED** [review]
LGTM

**Pull request merged**